draft-ietf-httpbis-digest-headers.md

```diff
@@ -419,6 +411,10 @@ Insecure digest algorithms MAY be used to preserve integrity against accidental
 NOT be used in a potentially adversarial setting; for example, when signing the digest of content for
 authenticity.
 
+When it is convenient to do so,
+the sender and the receiver can dynamically compute the checksum value
+while streaming the content.
+
```

Comment:
As an idea, what do you think about moving these 3 paragraphs to above the declaration of the registry? I could propose a follow up PR to this one if its easier

Reply:
@LPardue Great, thanks!

Reply:
I did it... PTAL